main.c

```diff
@@ -233,6 +233,14 @@ main(int argc, char *argv[])
 	/* Success! */
 	return (0);
 
+err1:
+	/* Zero and free the password. */
```

Comment:
I don't normally repeat comments like this in the error path -- it doesn't matter so much here, but it's often annoyingly repetitive when exactly the same comments appear in the "success" cleanup path too.